Scientist4JCore/src/main/java/com/github/rawls238/scientist4j/Experiment.java

```diff
@@ -59,77 +62,106 @@ public String getName() {
+        } catch (Exception e) {
+            observation.setException(e);
+        } finally {
+            observation.endTimer();
+            if (shouldThrow && observation.getException().isPresent()) {
+                throw observation.getException().get();
+            }
+            return observation;
         }
-        return observation;
-      }
     }
 
     protected boolean compareResults(T controlVal, T candidateVal) {
-      return controlVal.equals(candidateVal);
+        return controlVal.equals(candidateVal);
     }
 
     public boolean compare(Observation<T> controlVal, Observation<T> candidateVal) throws MismatchException {
-      boolean resultsMatch = !candidateVal.getException().isPresent() && compareResults(controlVal.getValue(), candidateVal.getValue());
-      totalCount.inc();
-      if (!resultsMatch) {
-        mismatchCount.inc();
-        handleComparisonMismatch(controlVal, candidateVal);
-      }
-      return true;
+        boolean resultsMatch = !candidateVal.getException().isPresent() && compareResults(controlVal.getValue(), candidateVal.getValue());
+        totalCount.inc();
+        if (!resultsMatch) {
+            mismatchCount.inc();
+            handleComparisonMismatch(controlVal, candidateVal);
+        }
+        return true;
     }
 
-    protected void publish(Result<T> r) {
-      return;
-    }
+    protected void publish(Result<T> r) {}
 
     protected boolean runIf() {
-      return true;
+        return true;
     }
 
     protected boolean enabled() {
-      return true;
+        return true;
+    }
+
+    protected boolean isAsync() {
+        return false;
     }
 
     private void handleComparisonMismatch(Observation<T> controlVal, Observation<T> candidateVal) throws MismatchException {
-      String msg;
-      if (candidateVal.getException().isPresent()) {
-        String stackTrace = candidateVal.getException().get().getStackTrace().toString();
-        String exceptionName = candidateVal.getException().get().getClass().getName();
-        msg = new StringBuilder().append(candidateVal.getName()).append(" raised an exception: ")
-                .append(exceptionName).append(" ").append(stackTrace).toString();
-      } else {
-        msg = new StringBuilder().append(candidateVal.getName()).append(" does not match control value (")
-                .append(controlVal.getValue()).append(" != ").append(candidateVal.getValue()).append(")").toString();
-      }
-      throw new MismatchException(msg);
+        String msg;
+        if (candidateVal.getException().isPresent()) {
+            String stackTrace = candidateVal.getException().get().getStackTrace().toString();
+            String exceptionName = candidateVal.getException().get().getClass().getName();
+            msg = new StringBuilder().append(candidateVal.getName()).append(" raised an exception: ")
+                    .append(exceptionName).append(" ").append(stackTrace).toString();
```

Comment:
all these spacing changes here too should go away, I'm sure these are just IDE settings